config/imsim/singleFrameDriver.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of obs_lsst.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+import os.path
+
+from lsst.utils import getPackageDir
+
+config.processCcd.load(os.path.join(getPackageDir("obs_lsst"), "config",
```

Comment:
I *think* this should load the `processCcd.py` from this package instead of the one from `obs_lsst`.

Reply:
Modified config/imsim/singleFrameDriver.py.. so that will work as long as IMAGEPROCESSINGPIPELINES_DIR is set.  I tried this out with w_2020_24 and can confirm that if I do something like `singleFrameDriver.py <repo> --show config --configfile=$PWD/ImageProcessingPipelines/config/imsim/singleFrameDriver.py` the resulting config is different than running without specifying a config file.  Such as config.processCcd.isr.doBrighterFatter=False vs config.processCcd.isr.doBrighterFatter=True

Reply:
Instead of this config file, maybe we can use the `processCcd.py` config explicitly when running `singleFrameDriver.py`:
```
singleFrameDriver.py <repo> --show config --configfile=$PWD/ImageProcessingPipelines/config/imsim/processCcd.py
```
assuming that does the right thing.  

Eventually, I think we should eups-ify this package, that way once it's set up, `IMAGEPROCESSINGPIPELINES_DIR` will be set.  eups-ifying this package would also allow us to move scripts in the `python/util` folder to `bin.src` and have those available via `$PATH`.  The eups-ifying project may better done in a separate PR.

Reply:
Deal - eups-ifying in another PR.. I'll open an issue.